app/workers/attachments/finish_direct_upload_job.rb

```diff
@@ -79,7 +79,7 @@ def validate_attachment(attachment, whitelist)
     contract = create_contract attachment, whitelist
 
     unless contract.valid?
-      errors = contracterrors.full_messages.join(", ")
+      errors = contract.full_messages.join(", ")
```

Comment:
wouldn't that be `contract.errors`?

Reply:
You are right. I should probably add a spec for this too.